config/webpack.test.js

```diff
@@ -87,84 +98,140 @@ module.exports = function (options) {
          *
          * See: https://github.com/webpack/source-map-loader
          */
-        // {
-        //   test: /\.js$/,
-        //   use: ['source-map-loader'],
-        //   exclude: [
-        //     // these packages have problems with their sourcemaps
-        //     helpers.nodeModulePath("mydatepicker"),
-        //     helpers.nodeModulePath("ng2-completer"),
-        //     helpers.nodeModulePath("angular2-flash-messages"),
-        //     helpers.nodeModulePath("ngx-dropdown"),
-        //     helpers.nodeModulePath("ngx-modal"),
-        //     helpers.nodeModulePath("ngx-modal"),
-        //     helpers.nodeModulePath("ng2-dnd")
-        //   ]
-        // },
+        {
+          test: /\.js$/,
+          use: ['source-map-loader'],
+          exclude: [
+            // these packages have problems with their sourcemaps
+            helpers.root('node_modules/rxjs'),
+            helpers.root('node_modules/@angular')
+            //     helpers.nodeModulePath("mydatepicker"),
```

Comment:
Are these commented lines intentional?

Reply:
this could be argued that it could be removed, honestly it was commented out to see if we could do it without it